src/views/SpatialDisplay.vue

```diff
@@ -127,9 +134,31 @@ export default class SpatialDisplay extends Mixins(WMSMixin) {
         }
       }
       groupNode?.children?.push(item)
+      if (layer.boundingBox) {
+        this.layersBbox[layer.name] = this.convertBoundingBoxToLatLngArray(layer.boundingBox)
+      }
     }
   }
 
+  private convertBoundingBoxToLatLngArray(boundingBox: BoundingBox): [number, number, number, number] {
+    const crs = boundingBox.crs
+
+    const minx = parseFloat(boundingBox.minx)
+    const miny = parseFloat(boundingBox.miny)
+    const maxx = parseFloat(boundingBox.maxx)
+    const maxy = parseFloat(boundingBox.maxy)
+
+    const p1 = toWgs84(point([minx, miny], { crs: crs }))
+    const p2 = toWgs84(point([maxx, maxy], { crs: crs }))
+
+    return [
```

Comment:
Have it return a [Mapbox LngLatBoundsLike]( https://docs.mapbox.com/mapbox-gl-js/api/geography/#lnglatboundslike), as that is the type that `fitBounds()` expects.